kubectl-plugin/pkg/cmd/job/job_submit.go

```diff
@@ -326,7 +326,7 @@ func (options *SubmitJobOptions) Run(ctx context.Context, factory cmdutil.Factor
 		}
 		clusterReady = isRayClusterReady(currCluster)
 		if !clusterReady {
-			err = fmt.Errorf("Cluster is not ready: %w", err)
+			err = fmt.Errorf("Cluster is not ready")
```

Comment:
Do we even need an error? 
```
fmt.Println("Cluster is not ready yet...")
```

Reply:
I think it gives visibility to what stage of the command is currently at. WDYT?

Reply:
by passing an error to `fmt.Println` instead of just inlining the string?

Reply:
Done!